slowquant/qiskit_interface/circuit_wavefunction.py

```diff
@@ -142,96 +138,78 @@ def __init__(
         # HF like orbital rotation indecies
         self.kappa_hf_like_idx = []
         for p in range(0, self.num_orbs):
             for q in range(p + 1, self.num_orbs):
                 if p in self.inactive_idx and q in self.virtual_idx:
-                    self.kappa_hf_like_idx.append([p, q])
+                    self.kappa_hf_like_idx.append((p, q))
                 elif p in self.inactive_idx and q in self.active_unocc_idx:
-                    self.kappa_hf_like_idx.append([p, q])
+                    self.kappa_hf_like_idx.append((p, q))
                 elif p in self.active_occ_idx and q in self.virtual_idx:
-                    self.kappa_hf_like_idx.append([p, q])
-        self._energy_elec: float | None = None
+                    self.kappa_hf_like_idx.append((p, q))
         # Setup Qiskit stuff
         self.QI = quantum_interface
         self.QI.construct_circuit(
             self.num_active_orbs, (self.num_active_elec // 2, self.num_active_elec // 2)
         )
 
     @property
-    def c_orthonormal(self) -> np.ndarray:
-        """Get orthonormalization coefficients (MO coefficients).
+    def kappa(self) -> list[float]:
+        """Get orbital rotation parameters."""
+        return self._kappa.copy()
 
-        Returns:
-            Orthonormalization coefficients.
-        """
-        return self._c_orthonormal
-
-    @c_orthonormal.setter
-    def c_orthonormal(self, c: np.ndarray) -> None:
-        """Set orthonormalization coefficients.
+    @kappa.setter
+    def kappa(self, k: list[float]) -> None:
+        """Set orbital rotation parameters.
 
         Args:
-            c: Orthonormalization coefficients.
+            k: orbital rotation parameters.
         """
         self._h_mo = None
         self._g_mo = None
         self._energy_elec = None
-        self._c_orthonormal = c
+        self._kappa = k.copy()
 
     @property
-    def c_trans(self) -> np.ndarray:
-        """Get orbital coefficients.
+    def c_mo(self) -> np.ndarray:
+        """Get molecular orbital coefficients.
 
         Returns:
-            Orbital coefficients.
+            Molecular orbital coefficients.
         """
-        kappa_mat = np.zeros_like(self._c_orthonormal)
+        kappa_mat = np.zeros_like(self._c_mo)
         if len(self.kappa) != 0:
-            if np.max(np.abs(self.kappa)) > 0.0:
-                for kappa_val, (p, q) in zip(self.kappa, self.kappa_idx):
-                    kappa_mat[p, q] = kappa_val
-                    kappa_mat[q, p] = -kappa_val
-        if len(self.kappa_redundant) != 0:
-            if np.max(np.abs(self.kappa_redundant)) > 0.0:
-                for kappa_val, (p, q) in zip(self.kappa_redundant, self.kappa_redundant_idx):
-                    kappa_mat[p, q] = kappa_val
-                    kappa_mat[q, p] = -kappa_val
-        return np.matmul(self._c_orthonormal, scipy.linalg.expm(-kappa_mat))
+            if np.max(np.abs(np.array(self.kappa) - np.array(self._kappa_old))) > 0.0:
```

Comment:
A comment to why we build the kappa matrix as difference values maybe? (sth about expansion point changes)
#readibility